enterprise/ha/src/main/java/org/neo4j/kernel/ha/transaction/TransactionPropagator.java

```diff
@@ -103,9 +103,12 @@ public SlavePriority getReplicationStrategy()
             {
                 switch ( config.get( HaSettings.tx_push_strategy ) )
                 {
-                    case fixed:
+                    case fixed_descending:
                         return SlavePriorities.fixed();
```

Comment:
Probably `SlavePriorities#fixed()` should be renamed to `SlavePriorities#fixedDescending()`